doc/user/backup_cronjob/README.md

```diff
@@ -0,0 +1,41 @@
+# Periodic etcd Backup using CronJob
+
+This doc talks about how to use Kubernetes [CronJob][k8s_cronjob] to make periodic etcd backups.
+
+Prerequisites:
+
+- Kubernetes cluster and kubectl ready
+- etcd-operator, etcd-backup-operator deployed
+- `example-etcd-cluster` EtcdCluster CR is created and the cluster is running
+
+All commands assume pwd in the root dir of etcd-operator repo.
```

Comment:
Maybe change `pwd in the root dir` to `the current working directory to be the root directory`.